src/libraries/System.DirectoryServices.Protocols/src/System.DirectoryServices.Protocols.csproj

```diff
@@ -10,19 +10,21 @@
     <AddNETFrameworkPlaceholderFileToPackage>true</AddNETFrameworkPlaceholderFileToPackage>
     <AddNETFrameworkAssemblyReferenceToPackage>true</AddNETFrameworkAssemblyReferenceToPackage>
     <PackageDescription>Provides the methods defined in the Lightweight Directory Access Protocol (LDAP) version 3 (V3) and Directory Services Markup Language (DSML) version 2.0 (V2) standards.</PackageDescription>
-    <!-- CS3016: Arrays as attribute arguments is not CLS-compliant -->
-    <NoWarn>$(NoWarn);CS3016</NoWarn>
     <EnableAOTAnalyzer>false</EnableAOTAnalyzer>
 
     <!-- TODO https://github.com/dotnet/runtime/issues/90400: Annotate for nullable reference types -->
     <Nullable>disable</Nullable>
-    <NoWarn>$(NoWarn);nullable</NoWarn>
+
+    <!-- CS3016: Arrays as attribute arguments is not CLS-compliant -->
+    <!-- CA2020: Prevent behavioral change caused by built-in operators of IntPtr/UIntPtr -->
```

Comment:
Can we use `nint` to avoid this warning? Or is there a problem with downlevel if we were to do that?

Reply:
Yes I believe we can just switch to that; it's only used in internal implementation and is a better fit since the field is not used as a pointer - thanks.